### PR TITLE
Standardise all compiler flags, fix bug for UNITY_4_7 exclusion

### PR DESCRIPTION
**Pull request by seaders**
Change checks to positive check if we're UNITY_5 instead of all the UNITY_4_x.

### CODE REVIEW

**Comment by seaders** [comment]
@Sergio-Mira could you take a quick look?

File: unity3d/Assets/Editor/SwrveBuildComponent.cs [hunk 1]
```diff
@@ -21,10 +21,10 @@ public class SwrveBuildComponent : SwrveCommonBuildComponent
     [MenuItem ("Swrve Demo/Export unityPackage")]
     public static void ExportUnityPackage ()
     {
-#if (UNITY_4_0 || UNITY_4_1 || UNITY_4_2 || UNITY_4_3 || UNITY_4_4 || UNITY_4_5 || UNITY_4_6)
-        EditorUserBuildSettings.SwitchActiveBuildTarget (BuildTarget.iPhone);
-#else
+#if (UNITY_5)
```
Comment (Sergio-Mira) [inline unity3d/Assets/Editor/SwrveBuildComponent.cs]: ~ Could we move it to:
#if UNITY_5

**Comment by Sergio-Mira** [comment]
:rocket:

File: unity3d/Assets/Editor/SwrveBuildComponent.cs (hunk 1 above)
Comment (seaders) [inline unity3d/Assets/Editor/SwrveBuildComponent.cs]: d'oh, of course!

**Pull request merged**